contributed_definitions/nyaml/NXlockin.yaml

```diff
@@ -1,111 +1,136 @@
 category: base
 doc: |
-  Base classes definition for lock in devices.
+  A base class definition for a lock-in amplifier.
+  The lock-in amplifier information: the device is being used to extract a (potentially)
```

Comment:
Suggest to have an empty line between the first and second lines here.

Reply:
noted to be implemented!